examples/time-log-kopia-repository-server/README.md

```diff
@@ -0,0 +1,231 @@
+```
+
+- Create Repository Server CR
+
+```bash
+vi repo-server-cr.yaml 
+```
+```
+apiVersion: cr.kanister.io/v1alpha1
+kind: RepositoryServer
+metadata:
+  labels:
+    app.kubernetes.io/name: repositoryserver
+    app.kubernetes.io/instance: repositoryserver-sample
+    app.kubernetes.io/part-of: kanister
+    app.kuberentes.io/managed-by: kustomize
+    app.kubernetes.io/created-by: kanister
+  name: kopia-repo-server-1
+  namespace: kanister
+spec:
+  storage:
+    secretRef:
+      name: s3-location
+      namespace: kanister
+    credentialSecretRef:
+      name: s3-creds
+      namespace: kanister
+  repository:
+    rootPath: /repo-controller/
+    passwordSecretRef:
+      name: repo-pass
+      namespace: kanister
+    username: kanisterAdmin
+    hostname: time-log.app
+  server:
+    adminSecretRef:
+      name: repository-admin-user
+      namespace: kanister
+    tlsSecretRef:
+      name: repository-server-tls-cert
+      namespace: kanister
+    userAccess:
+      userAccessSecretRef:
+        name: repository-server-user-access
+        namespace: kanister
+      username: kanisteruser
+```
+
+```bash
+$ kubectl create -f repo-server-cr.yaml -n kanister
+```
+**NOTE:**
+
+Make Sure the Repository Server is in ServerReady State before creating actionsets.
+You could check the status of the Repository Server CR by running following command
+```bash
+$ kubectl get repositoryservers.cr.kanister.io kopia-repo-server-1 -n kanister -o yaml
+```
+
+**NOTE:**
+
+The above command will configure a kopia repository server, which manages artifacts resulting from Kanister
+data operations such as backup.
+This is stored as a `repositoryservers.cr.kanister.io` *CustomResource (CR)* which is then referenced in Kanister ActionSets.
+
+#### Create Blueprint
+
+Create Blueprint in the same namespace as the Kanister controller
+
+```bash
+$ kubectl create -f ./examples/time-log/time-log-blueprint.yaml -n kanister
+```
+
+You can now take a backup of ``timelog`` application's data using an ActionSet defining backup for this application. Create an ActionSet in the same namespace as the controller.
+```
+# Create the actionset that causes the controller to kick off the backup
+$ kanctl create actionset --action backup --namespace kanister --blueprint time-log --deployment time-log/time-logger --repository-server kanister/kopia-repo-server-1
+
+actionset "s3backup-f4c4q" created
+
+# View the status of the actionset
+
+$ kubectl describe actionsets.cr.kanister.io -n kanister s3backup-f4c4q -oyaml
+```
+
+### 3. Restore the Application
+
+```bash
+$ kanctl -n kanister create actionset --action restore --from "s3backup-f4c4q"
+
+Warning: Neither --profile nor --repository-server flag is provided.
+Action might fail if blueprint is using these resources.
+actionset "restore-s3restore-g235d-23d2f" created
+
+# View the status of the actionset
```

Comment:
Same comment as above.
We should also mention that we have to wait for it to be completed.